dev/DataTarget/NameNormalizer.php

```diff
@@ -15,7 +15,8 @@ public static function normalize(string $key): string
             throw new TransliterationException();
         }
 
-        $key = str_replace([' ', ';', ',', '(', ')', '-', '.', '\'', '"', '/', '|', '=', '!', '?', '*', '[', ']', '~'], '_', $key);
+        $key = str_replace(['ʌ'], ['e'], $key);
```

Comment:
Did you try to fix the transliterator? I'm working on a package to add a user friendly interface for it, would be great if we managed a way to get all non-ASCII characters removed instead of maintaining a blacklist here. ;)

Reply:
> Did you try to fix the transliterator?

I am planning to publish a JSON object for transliterating missing IPA characters: https://github.com/szepeviktor/convert-ipa-characters-to-ascii

Reply:
> all non-ASCII characters removed

It is not transliteration's job to replace/remove writing signs.
My proposal is
1. Transliterate with ext-intl
2. Transliterate IPA characters from an array
3. Replace non-alphanumeric `/[0-9a-z_A-Z]/` characters with underscore
4. Check the result

Reply:
The documentation for the intl transliteration can be found here: https://unicode-org.github.io/icu/userguide/transforms/general/#script-transliteration

We could do something like:
```
[:^ASCII:] Remove
```

But I guess there's something missing with decompose settings where a simple NFKD could fix this. But I haven't found the time to look at that yet.

Reply:
I see you want to learn ICU and do everything during transliteration.

There is `IPA-XSampa` conversion in ICU 🤯